docs/test/replication-controller-list.md

```diff
@@ -0,0 +1,39 @@
+
+| Factor                               | 1                                                | 2                                             | 3                                        | 4                                       | 5                                                                 | 6                                                                   | Comment |
+|--------------------------------------|--------------------------------------------------|-----------------------------------------------|------------------------------------------|-----------------------------------------|-------------------------------------------------------------------|---------------------------------------------------------------------|---------|
+| Replication controller count on page | 1 replication controller on wide page            | 1 replication controller on narrow page       | 3 replication controller on on wide page | 3 replication controller on narrow page | Any other positive number of replication controllers on wide page | Any other positive number of replication controllers on narrow page |         |
+| Replication controller status        | Valid                                            | Invalid (for example with not-existing image) |                                          |                                         |                                                                   |                                                                     |         |
+| Replication controller data          | Image string length (short or too long for card) | Age                                           | Logs (with links)                        | Internal and external endpoints         | Number of running and desired pods                                | Labels                                                              |         |
+| Actions                              | View details                                     | Edit pod count                                | Delete                                   | Logs                                    |                                                                   |                                                                     |         |
+| Edit pod count action                | Non-positive value                               | 1                                             | Value less than 10                       | Value bigger than 30                    | String value                                                      | Empty field                                                         |         |
+| Delete replication controller action | Okay button                                      | Cancel button                                 | Delete service checkbox checked          | Delete service checkbox not checked     |                                                                   |                                                                     |         |
```

Comment:
Ok, I would shorten the text a bit. Also, there are much more test cases about this check box. Investigate a bit of the algorithm and find more levels